app/components/utils/add-signed-documents/component.js

```diff
@@ -4,11 +4,13 @@ import { inject } from '@ember/service';
 import { computed } from '@ember/object';
 import CONFIG from 'fe-redpencil/utils/config';
 import isAuthenticatedMixin from 'fe-redpencil/mixins/is-authenticated-mixin';
+import { tracked } from '@glimmer/tracking';
 
 export default Component.extend(UploadDocumentMixin, isAuthenticatedMixin, {
   store: inject(),
   classNames: ['vl-u-spacer'],
   isAddingDocument: null,
+  @tracked isAddingNewDocument: null,
```

Comment:
:thinking: Kan het dat je deze niet nodig hebt? `tracked` is deel van de glimmer api, en dit is nog geen glimmer component (zie ... `import Component from '@ember/component';` vs `import Component from '@glimmer/component';`)Volgens mij zijn alle properties van een Ember component "tracked"?

Reply:
Net getest en werkt inderdaad nog correct, ik haal het weg.